saleor/graphql/webhook/subscription_payload.py

```diff
@@ -18,7 +18,9 @@
 logger = get_task_logger(__name__)
 
 
-def initialize_request(requestor=None, sync_event=False) -> SaleorContext:
+def initialize_request(
+    requestor=None, sync_event=False, is_mutation=False
```

Comment:
I would change `is_mutation=False` to `allow_replica=True` everywhere to keep it consistent.

Reply:
You are right, this function is only used by subscription stuff.